pkg/resources/aws/aws.go

```diff
@@ -1324,6 +1324,10 @@ func FindNatGateways(cloud fi.Cloud, routeTables map[string]*resources.Resource,
 		}
 		response, err := c.EC2().DescribeNatGateways(request)
 		if err != nil {
+			if awsup.AWSErrorCode(err) == "NatGatewayNotFound" {
+				klog.V(2).Infof("Got NatGatewayNotFound describing NatGateways; will treat as already-deleted")
```

Comment:
I think this is or may be used in the future in some other places also. 
```suggestion
				klog.Warningf("Got NatGatewayNotFound describing NatGateways; will treat as deleted")
```

Reply:
All the other "will treat as already-deleted" log lines are at `V(2).Infof`, why should this one be at `Warningf`?

Reply:
Didn't see those other ones, sorry. Please ignore.